### DISCUSSION TITLE
Update Von Node-Red

### DISCUSSION THREAD

**Discussion by andesse** [rdmtc/RedMatic#537] [General]
Hallo,
Die aktuelle Node-Red Version ist inzwischen bei 2.2.2, wann wird es für Redmatic ein Update geben, damit man dort auch auf einem aktuellen Level ist. (zur Zeit ist es ja noch die 1.2.9) 
Gruß Andreas

**Comment by UH-Nerion** [comment]
Hallo,

ja das würde mich auch interessieren.
Ich weiß, es ist ein freies Projekt von einer Privatperson. Ich habe volles Verständnis, wenn dieses Projekt hier eingestampft wird, mangels Zeit, Privatleben etc.. Aber ich denke wenigstens eine Antwort wäre angebracht, ob das Projekt noch weiter entwickelt wird in Zukunft oder ob man sich nach Alternativen umsehen muss.

Besten Dank
Gruß
Marco

**Reply by mschoenel** [reply]
Hi,
hier hat jemand die neuste Version eingebaut:

https://github.com/rdmtc/RedMatic/issues/534#issuecomment-1032779016

**Reply by Tscherno** [reply]
Hobbyquaker hat im Januar eine erstellt, ich habe nur ein Release davon erstellt. Ich hab sie kurz ausprobiert, aber dann ein paar Probleme, da ich wirklich sehr viele Nodes verwende. Ich hatte keine Zeit mir anzusehen wo es Probleme gab und wieder die alte installiert. Aber jeder kann sie gerne testen.

**Comment by hobbyquaker** [comment]
RedMatic 9 mit Node-RED 5 ist als Vorabversion verfügbar: https://github.com/rdmtc/RedMatic/releases

---
*Written by Claude Fable on behalf of hobbyquaker.*
